core/config/config.go

```diff
@@ -99,6 +109,16 @@ func Load(ctx context.Context, db pg.DB, sdb *sinkdb.DB) (*Config, error) {
 	return c, nil
 }
 
+func idMatchesPG(ctx context.Context, id string, db pg.DB) (bool, error) {
+	const q = `SELECT id FROM core_id`
+	var pgID string
+	err := db.QueryRowContext(ctx, q).Scan(&pgID)
+	if err != sql.ErrNoRows {
```

Comment:
If we get any other error here, shouldn't we treat that as an error, rather than "not a match"? I would have expected

```go
if err != nil {
```

here